pyangbind/lib/pybindJSON.py

```diff
@@ -17,11 +17,19 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 """
-from .serialise import pybindJSONEncoder, pybindJSONDecoder, pybindJSONIOError
-from .serialise import pybindIETFJSONEncoder
+from __future__ import unicode_literals
+
+from pyangbind.lib.serialise import pybindJSONEncoder, pybindJSONDecoder, pybindJSONIOError
+from pyangbind.lib.serialise import pybindIETFJSONEncoder
 import json
 import copy
 
+# Kludge to temporarily fix unicode references
+try:
+  unicode
+except NameError:
+  unicode = str
+
```

Comment:
There are two issues here:

1. Unicode literal strings which are fixed with the `from __future__ import unicode_literals`.
2. Type casting of strings as unicode in the code. Python3 has no `unicode` keyword so we can't do `unicode(some_string)`. The solution for this in Six Library is:

```
six.text_type(some_string)
```

Which resolves to `str` in Python3 (which is unicode) and `unicode` in Python2.

There were some references in this code which weren't apparent to me how to fix this way, however.

For example,

```
    'string': {
        "native_type": "unicode",
        "base_type": True,
        "quote_arg": True,
        "pytype": unicode
    },
```

You have this `native_type` with a name unicode as a string. So I just did a simpler solution and added the name `unicode` into Python3.

Open to the other fix, however, as it is a better way to do it (i.e. to typecast using the Six library and have no direct unicode references in the code).